poleno/utils/static/poleno/js/collapsable.js

```diff
@@ -0,0 +1,61 @@
+/* Enables elements to automatically show or hide other elements by bootstrap collapsing.
+ * Works on all elements type, which must contain data-toggle="collapse" and
+ * data-target="`css-selector`" attributes. Collapsable element must contain class="collapse".
+ *
+ * data-target-shown elements are shown, when collapsable elements are hidden. data-target-hidden
+ * elements are shown, when collapsable elements are shown and vice versa.
+ *
+ * By entering url with a hash into browser, content and elements will be automatically collapsed.
+ *
+ * Requires:
+ *  -- JQuery
+ *
+ * Examples:
+ *  <div class="parent">
+ *    <div data-toggle="collapse" data-target="#content">
```

Comment:
@martinmacko47 Priklad: Ak **X** ma `data-target` na collapse **A, B**, a **Y** ma collapse iba na **B**.  Pri skrytom **A** a uz rozbalenom **B** a zakliknuti na **X** nastane toggle elementov **A a B**. Je to zrejme nechcenne spravanie. Budeme ale realne niekde v projekte takto pouzivat collapse?

Reply:
@viliambalaz IMHO taku situaciu nebudeme mat. Mali by sme vsade mat iba taketo jednoduche pouzitia. Ze ak sa element A collapsuje naraz s elementom Y, tak vzdy budu oba spolu.